frigate/detectors/detector_config.py

```diff
@@ -51,6 +51,9 @@ class ModelConfig(BaseModel):
     model_type: ModelTypeEnum = Field(
         default=ModelTypeEnum.ssd, title="Object Detection Model Type"
     )
+    order: list[int] = Field(
```

Comment:
this config would apply to every detector so it needs to be used for every detector or perhaps the default value should be `None` and the field changes to be called `custom_model_output_order` or something